### PR TITLE
Simple docker-compose.yml

### PR DESCRIPTION
**Pull request by TheNexter**
ready to merge docker-compose.yml + readme update with docker-compose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,25 @@
...
 
+#### Docker Compose
+```
+version: "2.1"
+services:
+  memos:
+    image: neosmemo/memos:latest
+    container_name: memos
+    volumes:
+      - /path/to/data:/var/opt/memos #Path where your notes are saved
```
Comment (boojack) [inline README.md:42]: Same as `docker run`.
```suggestion
      - ~/.memos/:/var/opt/memos
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: "2.1"
+services:
+  memos:
+    image: neosmemo/memos:latest
+    container_name: memos
+    volumes:
+      - /path/to/data:/var/opt/memos #Path where your notes are saved
```
Comment (boojack) [inline docker-compose.yml:7]:
```suggestion
      - ~/.memos/:/var/opt/memos
```

File: README.md [hunk 3]
```diff
@@ -26,10 +26,25 @@
...
 ## Deploy with Docker
 
+#### Docker Run
 ```docker
 docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest
 ```
 
+#### Docker Compose
+```
+version: "2.1"
```
Comment (boojack) [inline README.md:36]: Update version to `3`
```suggestion
version: "3"
```

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+version: "2.1"
```
Comment (boojack) [inline docker-compose.yml:1]:
```suggestion
version: "3"
```

**Comment by TheNexter** [comment]
It's update

File: README.md [hunk 5]
```diff
@@ -26,10 +26,25 @@
 
 ## Deploy with Docker
 
+#### Docker Run
```
Comment (boojack) [inline README.md:29]:
```suggestion
#### Docker Run

```

File: README.md [hunk 6]
```diff
@@ -26,10 +26,25 @@
...
 ```docker
 docker run -d --name memos -p 5230:5230 -v ~/.memos/:/var/opt/memos neosmemo/memos:latest
 ```
 
+#### Docker Compose
```
Comment (boojack) [inline README.md:34]:
```suggestion
#### Docker Compose

```

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**